postOffice2.py

```diff
@@ -0,0 +1,148 @@
+class Message:
+    """
+    Class representing a message
+    Parameters
+    ----------
+    sender : str
+        The message sender's username.
+     message_body : str
+        The body of the message.
+    title: str
+        title of the message
+    message_id : int
+        Incremental id of the last message sent.
+
+    Attributes
+    ----------
+    sender : str
+        The message sender's username.
+    message_body : str
+        The body of the message.
+    title: str
+        title of the message
+    message_id : int
+        Incremental id of the last message sent.
+    seen : bool
+        message already seen or not
+    """
+    def __init__(self, sender, id, title, body):
```

Comment:
Use type hints and return types -
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/